api/src/test/java/org/openmrs/ChemoDosingInstructionsTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License, 
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under 
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS 
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.util.Locale;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.test.BaseContextSensitiveTest;
+
+public class ChemoDosingInstructionsTest extends BaseContextSensitiveTest {
+
+	private static final String DILUTION_INSTRUCTIONS_CONSTANT = "10% diluted";
+
+	private static final Double ADJUSTMENT_PERCENTAGE_CONSTANT = 20.04;
+
+	private static final String DELIVERED_TIMING_CONSTANT= "once in 2 hours";
+
+	private static final String ADJUSTMENT_PERCENTAGE = "Dosage Adjustment Percentage : ";
+
+	private static final String DELIVERED_TIMING = "Dosage Delivered Timing : ";
+
+	private static final String DILUTION_INSTRUCTIONS = "Dosage Dilution Instructions : ";
+
+	@Test
+	public void getDosingInstructionsAsString_shouldInferCorrectly() {
+		ChemoDosingInstructions chemoDosingInstructions = new ChemoDosingInstructions();
```

Comment:
@varung-31  I see you using the `ChemoDosingInstructions` instance in all your test cases, why do we just put it in a `setup()` method